io/io/inc/TFileMerger.h

```diff
@@ -55,6 +55,8 @@ class TFileMerger : public TObject {
    TList          fMergeList;                 ///< list of TObjString containing the name of the files need to be merged
    TList          fExcessFiles;               ///<! List of TObjString containing the name of the files not yet added to fFileList due to user or system limitation on the max number of files opened.
 
+   bool fOutFileWasExplicitlyClosed = false; ///<! the user has called CloseOutputFile(), so we shouldn't error out in RecursiveRemove
```

Comment:
Since we're adding a new data member we probably need to increase the class version

Reply:
My bad, I didn't notice that the data member is transient. All good then.